.github/workflows/node.yml

```diff
@@ -0,0 +1,20 @@
+name: Node
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
```

Comment:
(untested)

```suggestion
on:
  push:
    branches: [main]
    paths: "nitro-protocol/**"
  pull_request:
    paths: "nitro-protocol/**"

```

Reply:
I test this manually, it seems to work [5328ee2](https://github.com/statechannels/go-nitro/pull/487/commits/5328ee20430c13101b5693fd6dff08368dd09ea6) !